Automattic/wc-calypso-bridge
# Disable ecommerce menu and relevant JS fixes when SSO is disabled

Fixes #923 

### Specification

This PR adds detection for Jetpack SSO to disable all menu fixes in the JS. 

There are two issues here:
- When navigating to Calypso home, the "My Home" menu item isn't selected as active: 

<img width="2247" alt="Screenshot 2023-07-07 at 2 59 07 PM" src="https://github.com/Automattic/wc-calypso-bridge/assets/1105590/b6546b37-d859-4adf-8a27-2805dbcd7ea7">

- When navigating to the WC Admin home in wp-admin, the "My Home" and the "WooCommerce" menus are both active.

<img width="1857" alt="Screenshot 2023-07-07 at 3 00 48 PM" src="https://github.com/Automattic/wc-calypso-bridge/assets/1105590/00f3f203-8dde-49fe-9dc8-b19c5b9df578">

### Validation

Steps for testing:
- Navigate to **Settings > Security** and disable the "Allow users to log in to this site using WordPress.com accounts" option.
- (In a new tab) Navigate to WooCommerce Home. 
- Notice that the menu is working and is highlighted as expected based on the screenshot above.
- Now navigate to `wordpress.com/home/[siteslug_with_sso_disabled]` 
- Notice that the "My home" is highlighted as expected based on the screenshot above.
- Also, notice that the ecommerce menu is not applied to any context.
